stake/src/component.rs

```diff
@@ -580,6 +635,8 @@ impl Staking {
                     ?params.missed_blocks_maximum,
                     "recorded vote info"
                 );
+                // TODO: We may want to update or clear this metric for non-active validators.
```

Comment:
I think setting this to `0` as the validator moves out of `Active` state is a good idea.